content/posts/newsletter/2018-11-30.md

```diff
@@ -0,0 +1,102 @@
+which details how XMPP server administrators have access to user's metadata and
+other sensitive information.
+
+The German email-provider mailbox.org, has [unveiled a new revamped website](https://mailbox.org/en/post/a-new-logo-a-new-website-new-software-versions-and-new-features-at-mailbox-org)
+with new features, including migration to a new XMPP chat server (Ejabberd)
+and deployment of [Converse.js](http://conversejs.org/) as webchat.
+
+German IT website Golem.de has written about Daniel Gultsch's new free Android app
+[Quicksy](https://play.google.com/store/apps/details?id=im.quicksy.client)
+which allows users to sign up with their Cellphone number.
+Here's the [Google machine-translated
+version](https://translate.google.com/translate?hl=&sl=de&tl=en&u=https%3A%2F%2Fwww.golem.de%2Fnews%2Fquicksy-mit-der-telefonnummer-ins-jabber-netz-1811-137855.html)
+and the [original German article](https://www.golem.de/news/quicksy-mit-der-telefonnummer-ins-jabber-netz-1811-137855.html).
+
+The Stardust blog explains how you can [Write an XMPP bot in half an hour](https://starbeamrainbowlabs.com/blog/article.php?article=posts%2F327-Write-an-XMPP-bot-in-half-an-hour.html)
+
+Andrea Schäfer provided a [technical update and demo of Chatty](https://puri.sm/posts/librem5-progress-report-19/),
+the XMPP-capable chat client that is being developed for the upcoming Purism phone.
+This was posted already a while ago, but only came to my attention recently.
+
+<form style="padding: 10px; text-align:center; margin-bottom: 30px;"
+      action="https://tinyletter.com/xmpp" method="post" target="popupwindow"
+      onsubmit="window.open('https://tinyletter.com/xmpp', 'popupwindow',
+      'scrollbars=yes,width=800,height=600');return true">
+<p><label for="tlemail">Subscribe to receive the next edition in your inbox</label></p>
+<p><input type="text" placeholder="Email address" name="email" id="tlemail" /></p>
+<input type="hidden" value="1" name="embed"/>
+<input type="submit" style="padding: 10px; border-radius: 5%" value="Subscribe" />
+</form>
+
+## Upcoming Events
+
+The [23rd XMPP Summit has been announced](https://xmpp.org/2018/11/xmpp-summit-23/).
+It will be held on the 31st of January and 1st of February 2019 in Brussels,
+Belgium. These are the two days preceding FOSDEM 2019.
+
+## Software releases
+
+### Servers
+
+* [MongooseIM 3.2](https://www.erlang-solutions.com/blog/mongooseim-3-2-meet-our-inbox.html)
+* [Prosody 0.10.3](https://blog.prosody.im/prosody-0-10-3-released/), [0.11](https://blog.prosody.im/prosody-0-11-0-released/) and [0.11.1](https://blog.prosody.im/prosody-0-11-1-released/)
+
+### Clients
+
+* [Converse 4.0.5](https://github.com/conversejs/converse.js/releases/tag/v4.0.5)
+* [Gajim 1.1.0](https://dev.gajim.org/gajim/gajim/blob/gajim-1.1.0/ChangeLog)
+* [Opa 0.1.0](https://github.com/credija/opa/releases/tag/0.1.0) - A new webchat client based on Vue, NuxtJS and ElementUI.
```

Comment:
Spellings are: Vue.js, Nuxt.js, Element

Reply:
This was the spellings used in the README of Opa.

`Element` is way to generic, but I'll change the other two.